UWP/UWPHelpers/StoragePickers.cpp

```diff
@@ -0,0 +1,74 @@
+// UWP STORAGE MANAGER
+// Copyright (c) 2023 Bashar Astifan.
+// Email: bashar@astifan.online
+// Telegram: @basharastifan
+
+#include "pch.h"
+
+#include "StorageAsync.h"
+#include "StorageAccess.h"
+
+using namespace Platform;
+using namespace Windows::Storage;
+using namespace Windows::Foundation;
+
+extern void AddItemToFutureList(IStorageItem^ folder);
+
+// Call folder picker (the selected folder will be added to future list)
+concurrency::task<Platform::String^> PickSingleFolder()
+{
+	auto folderPicker = ref new Windows::Storage::Pickers::FolderPicker();
+	folderPicker->SuggestedStartLocation = Windows::Storage::Pickers::PickerLocationId::Desktop;
+	folderPicker->FileTypeFilter->Append("*");
+
+	return concurrency::create_task(folderPicker->PickSingleFolderAsync()).then([](StorageFolder^ folder) {
+		auto path = ref new Platform::String();
+	    if (folder != nullptr)
+	    {
+			AddItemToFutureList(folder);
+	    	path = folder->Path;
```

Comment:
Still some mixing of spaces and tabs in this file.  I recommend searching for "    " and cleaning up that way.  Should be quick.

-[Unknown]

Reply:
Sorry for asking this, but I'm a bit confused from the formating, in VS it look like below
![Capture](https://user-images.githubusercontent.com/3244951/235406992-cd4c5d52-cc7b-4541-b0af-2cc83ce11492.PNG)

is there any tool can help to fix these before submitting to GitHub?

Reply:
You can press Ctrl-R, then Ctrl-W to show whitespace indicators.  I usually search for a tab and it highlights where the indentation is funny.  But that said, most of the time I just don't have this problem, except maybe when pasting code from outside or something.

-[Unknown]